07-HTML5 APIs/canvas.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html ng-app="canvas">
+<head>
+	<title>Topic 7 - Canvas</title>
+
+	<script src="node_modules/angular/angular.min.js"></script>
```

Comment:
Falling in love with angular? :) This was not required for this topic but not a bad thing either.